tests/common/dhcp/test_dhcp.py

```diff
@@ -65,7 +65,18 @@ def test_wireserver_route_not_exists(self):
         self.assertTrue(dhcp_handler.gateway is None)
 
         # execute
-        self.assertFalse(dhcp_handler.wireserver_route_exists)
+        routing_table = "\
```

Comment:
NIT: Since this is being used in multiple places, why not make it a small helper function for this class? (Just trying to avoid code-duplication as much as possible)

Reply:
Hmm, that makes sense. Do you think it would be easiest to make two member variables? One with wireserver's info, and one without?

Reply:
You can do that, or maybe make the string without the wireserver info a template and add the line with the wireserver info if a flag is set? Totally your call, whatever you feel more comfortable with and you think is the easiest way to do :)